src/dialog/scss/variable.scss

```diff
@@ -122,3 +122,20 @@ $dialog-close-color-hovered: $color-text1-4 !default;
 /// background
 /// @namespace statement/hover/close
 $dialog-close-bg-hovered: $color-transparent !default;
+
+
```

Comment:
you can remove line128-141 and add the following code after line36
```
/// padding (t)
/// @namespace size/content
$dialog-message-title-padding-top: $s-5 !default;

/// padding (b)
/// @namespace size/content
$dialog-message-title-padding-bottom: $s-5 !default;

/// padding (l, r)
/// @namespace size/content
$dialog-message-title-padding-left-right: $s-5 !default;
```
1. I don't think we need the 'Alert'.
2. the namespace can be size/content, because it's not title, it's body or content in fact.